docs/source/support/component-statuses.rst

```diff
@@ -0,0 +1,329 @@
+.. SPDX-License-Identifier: Apache-2.0
+
+Feature and Component Statuses
+==============================
+
+This page gives an overview of the statuses of released components and features according to :doc:`status-definitions`. Anything not listed here implicitly has status "Labs", but it's possible that something accidentally slipped the list so if in doubt, please :doc:`contact us <support>`.
+
+Ledger API
+----------
+
+.. list-table::
+   :widths: 80 10 10
+   :header-rows: 1
+
+   * - Component/Feature
+     - Status
+     - Deprecated on
+   * - :doc:`Ledger API specification </app-dev/grpc/proto-docs>` including all semantics of >= DAML-LF 1.6
+     - Stable
+     -   
+   * - CLI and test names of :doc:`/tools/ledger-api-test-tool/index`
+     - Beta
+     -
+   * - `Numbered (ie non-dev) Versions of Proto definitions distributed via GitHub Releases <protobufs_>`_
+     - Stable
+     - 
+   * - `Dev Versions of Proto definitions distributed via GitHub Releases <protobufs_>`_
+     - Alpha
+     - 
+
+.. _runtime-components:
+
+Runtime components
+------------------
+
+.. list-table::
+   :widths: 80 10 10
+   :header-rows: 1
+
+   * - Component / Feature
+     - Status
+     - Deprecated on
+   * - **JSON API**
+     - 
+     -
+   * - HTTP endpoints under ``/v1/`` including status codes, authentication, query language and encoding.
+     - Stable
+     -
+   * - ``daml json-api`` CLI :ref:`for development <dev-tools>`. (as specified using ``daml json-api --help``)
+     - Stable
+     -
+   * - Stand-alone distribution for production use, including CLI specified in ``--help``.
+     - Stable
+     -
+   * - **Triggers**
+     -
+     -
+   * - :doc:`DAML API of individual Triggers </triggers/index>`
+     - Beta
+     -
+   * - :doc:`Development CLI to start individual triggers in dev environment </triggers/index>` (``daml trigger``)
+     - Beta
+     -
+   * - :doc:`/tools/trigger-service` (``daml trigger-service``)
+     - Alpha
+     -
+   * - **Extractor**
+     -
+     -
+   * - :doc:`/tools/extractor` (``daml extractor``)
+     - Labs
+     -
+
+.. _libraries:
+
+Libraries
+---------
+
+.. list-table::
+   :widths: 80 10 10
+   :header-rows: 1
+
+   * - Component / Feature
+     - Status
+     - Deprecated on
+   * - **Scala Ledger API Bindings**
+     -
+     -
+   * - ``daml codegen scala`` :doc:`CLI and generated code </app-dev/bindings-scala/index>`
+     - Stable
+     - 
+   * - ``bindings-scala_2.12`` :doc:`library and its public API </app-dev/bindings-scala/index>`
+     - Stable
+     - 
+   * - **Java Ledger API Bindings**
+     - 
+     -
+   * - ``daml codegen java`` :doc:`CLI and generated code </app-dev/bindings-java/codegen>`
+     - Stable
+     -
+   * - ``bindings-java`` :doc:`library and its public API </app-dev/bindings-java/index>`.
+     - Stable
+     -
+   * - ``bindings-rxjava`` :doc:`library and its public API </app-dev/bindings-java/index>`.
```

Comment:
the way this is structured, there isn’t an obvious way to say that we keep the rxjava bindings but deprecate the reactive components.

Reply:
I know. When we deprecate small units of functionality, I'll need to split them out. But I can't split out everything to that level of granularity to start with.

Reply:
Now split out and deprecated.